S/Skia/build_tarballs.jl

```diff
@@ -0,0 +1,105 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Skia"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+
+   GitSource("https://github.com/google/skia.git", "482de011c920d85fdbe21a81c45852655df6a809"),
+   GitSource("https://github.com/stensmo/cskia.git", "500cdca61e2105555f339fa363f55e30696b009f"),
+   DirectorySource("./bundled")
+
+]
+
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+
+
+platforms = [
+     Platform("x86_64", "linux"; libc = "glibc"),
+     Platform("aarch64", "linux"; libc = "glibc"),
+     Platform("riscv64", "linux"; libc = "glibc"), 
+     Platform("powerpc64le", "linux"; libc = "glibc"), 
+     Platform("i686", "linux"; libc = "glibc"),
+]
```

Comment:
So basically all Linux variants with glibc? But without 32bit ARM (is that intentional?) How about
```suggestion
platforms = supported_platforms()
filter!(p -> Sys.islinux(p) && libc(p) == "glibc", platforms)
```
which has the advantage of automatically getting support for further archs if needed.